src/components/User/CombinedVoucher.vue

```diff
@@ -84,7 +84,13 @@ export default{
             }else{
                 this.price=item.price
                 this.point=item.point
-                this.$router.push({name:"rewardpage",query:{id:this.id,shopId:id,name:name,voucher:item}})
+                item.imagename=image
+                this.currvoucher.push(item)
+                db.firestore().collection('users').doc(this.id).update({
+                    currvoucher:this.currvoucher,
+                    points:this.score-item.point
+                }).then(() => {this.$router.push({name:"rewardpage",query:{id:this.id,shopId:id,name:name,voucher:item}})})
```

Comment:
I just realized... I think for this one should go to 'voucherverify' instead of 'rewardpage'